app.rb

```diff
@@ -90,5 +91,5 @@ def verify_signature(payload_body)
 end
 
 def redmine_issue_repos
-  ENV['REDMINE_ISSUE_REQUIRED_REPOS'].to_s.split.map { |r| Regexp.new("\\A#{r}\\z") }
+  YAML.load_file('config/redmine_issue_required_repos.yaml')
```

Comment:
Or in the YAML, use `!ruby/regexp '/foo/'` where a regexp is actually needed perhaps..